### ISSUE TITLE
Unexpected value 'MultiPickerModule' imported by the module 'AppModule'

### ISSUE THREAD

**Issue by jaypatel16** [raychenfj/ion-multi-picker#24]
I am getting Unexpected value 'MultiPickerModule' imported by the module 'AppModule' when I import MultiPickerModule in NgModule imports:

```
@NgModule({
  declarations: [
    MyApp,
    pages,
    components,
    directives
  ],
  imports: [
    IonicModule.forRoot(MyApp, {
   //   tabsPlacement: 'top',
      tabsHideOnSubPages: 'true',
      platforms: {
          ios: {
            statusbarPadding: true
          }
      }
    }),
    ChartModule,
    CalendarModule,
    MultiPickerModule    
  ],
  bootstrap: [IonicApp],
  entryComponents: [
    MyApp,
    pages,
    components
  ],
  providers: [CacheService, HttpService],
})
```

Configuration:
Ionic - Angular : rc3
Angular : 2.0.0-beta.17
Npm : 3.10
Node : 6.3

**Comment by raychenfj** [comment]
Can't see anything obvious wrong in your snippt, here is mine. 
```
import { NgModule } from '@angular/core';
import { IonicApp, IonicModule } from 'ionic-angular';
import { MyApp } from './app.component';
import { SimpleExamplePage } from '../pages/simple/simple';
import { AdvancedExamplePage } from '../pages/advanced/advanced';
import { TabsPage } from '../pages/tabs/tabs';

import { MultiPickerModule } from 'ion-multi-picker';


@NgModule({
  declarations: [
    MyApp,
    SimpleExamplePage,
    AdvancedExamplePage,
    TabsPage
  ],
  imports: [
    IonicModule.forRoot(MyApp),
    MultiPickerModule
  ],
  bootstrap: [IonicApp],
  entryComponents: [
    MyApp,
    SimpleExamplePage,
    AdvancedExamplePage,
    TabsPage
  ],
  providers: []
})
export class AppModule {}
```

By the way, your angular version is old, currently ionic need ver 2.1.1. 

And please check your ionic cli version as well, make sure it's latest, and you may want to create a demo project with latest cli and then check your project structure as well. When ionic update , the project structure may change as well.

**Comment by jaypatel16** [comment]
Solved by Updating angular to 2.1.1

Thank you @raychenfj
